README.md

```diff
@@ -482,25 +482,30 @@ to perform TypeScript compilation. If you are bundling or otherwise modifying
 the layout of the `playground-elements` NPM package, you may need to add special
 handling for this file.
 
-This worker script must exist at `./playground-typescript-worker.js` in the same
-directory as the JavaScript file that contains the definition of
-`<playground-project>`, and is resolved relative to `import.meta.url`.
-
 ### Rollup
 
-Use the
+Use the Rollup
 [`@web/rollup-plugin-import-meta-assets`](https://github.com/modernweb-dev/web/tree/master/packages/rollup-plugin-import-meta-assets#readme)
 plugin to automatically copy the worker script into the correct location. See
 [examples/rollup](https://github.com/PolymerLabs/playground-elements/tree/main/examples/rollup)
 for an example configuration.
 
 ### Webpack
 
-Webpack 5+ [supports](https://webpack.js.org/guides/web-workers/) loading Web
-Workers with no additional plugins. See
+Webpack 5+ automatically [supports](https://webpack.js.org/guides/web-workers/)
+loading Web Workers with no additional plugins. See
```

Comment:
put link around all of "supports loading Web Workers"?

Reply:
Done.